tests/e2e/config/bootstrap.js

```diff
@@ -189,6 +189,7 @@ function observeConsoleLogging() {
 		if (
 			text.startsWith( 'Powered by AMP' ) ||
 			text.startsWith( 'data_error unknown response key' ) ||
+			text.startsWith( 'You are probably offline.' ) ||
```

Comment:
One small tweak: let's move this to a new section because the above comment isn't really appropriate.

I'll make the move, as this is otherwise good to merge 😄